inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_tests_common.hpp

```diff
@@ -70,6 +71,8 @@ class DSR_TestsCommon : virtual public LayerTestsUtils::LayerTestsCommon {
             results.emplace_back(std::make_shared<ngraph::opset3::Result>(output));
         }
 
+        results.insert(results.end(), m_additionalResults.begin(), m_additionalResults.end());
```

Comment:
Why did you need this?

Reply:
I've taken a look once more and I've found that it's not a good idea to use `DSR_TestsCommon` because I actually doesn't use it's features, so I've changed these classes to be based on `TestsCommon`, so this change is not needed anymore.